### PR TITLE
Illusivenetworks fix readme file

### PR DESCRIPTION
**Pull request by adiozer**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ X] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X ] No

## Must have
- [ ] Tests
- [X ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py [hunk 1]
```diff
@@ -628,6 +649,158 @@ def get_event_incident_id_command(client: Client, args: dict) -> Tuple:
...
+        file_results.append(fileResult(
+            filename=f'eventId_{event_id}_{artifact_type}_{i}.jpg',
+            data=zip_file.read(info.filename),
+            file_type=entryTypes["image"]
+        ))
+        file_names.append(f'eventId_{event_id}_{artifact_type}_{i}.jpg')
+    return file_results, file_names
+
+
+def link_forensics_artifacts_name_command(file_names, client: Client, args: dict) -> CommandResults:
```
Comment (guyfreund) [inline Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py:779]:
```suggestion
def link_forensics_artifacts_name_command(file_names: list, client: Client, args: dict) -> CommandResults:
```

File: Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py [hunk 2]
```diff
@@ -628,6 +649,158 @@ def get_event_incident_id_command(client: Client, args: dict) -> Tuple:
...
+            'Artifacts': file_names
+        }
+
+        return CommandResults(
+            outputs_prefix='Illusive.Event',
+            outputs_key_field='eventId',
+            outputs=outputs
+        )
+    else:
+        readable_output = f'### event id {event_id} has no artifacts'
```
Comment (guyfreund) [inline Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py:794]:
```suggestion
        readable_output = f'### Event ID {event_id} has no artifacts'
```

File: Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml [hunk 3]
```diff
@@ -516,7 +519,133 @@ script:
     - contextPath: Illusive.Event.status
       description: The status command ( Done, InProgress)
       type: String
-  dockerimage: demisto/python3:3.8.2.6981
+  - arguments:
+    - default: false
+      description: The incident ID on which to retrieve the associated events
+      isArray: false
+      name: incident_id
```
Comment (guyfreund) [inline Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml:526]: Where is this ID can be retrieved from? Please edit the description to include the command to run to get the ID from

File: Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml [hunk 4]
```diff
@@ -516,7 +519,133 @@ script:
...
+    - default: false
+      description: The event ID to retrieve the forensics artifacts from
+      isArray: false
+      name: event_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: DESKTOP_SCREENSHOTS
+      description: 'The type of forensics artifact to retrieve '
```
Comment (guyfreund) [inline Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml:637]: If `DESKTOP_SCREENSHOTS` is the only option available and it is also the default value, why do we need this argument?

File: Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml [hunk 5]
```diff
@@ -527,5 +656,5 @@ script:
   type: python
 tests:
 - No tests
```
Comment (guyfreund) [inline Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml:658]:
```suggestion
  - IllusiveNetworks-Test
```

File: Packs/IllusiveNetworks/ReleaseNotes/1_0_2.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+
+#### Playbooks
+##### Illusive-Incident-Escalation
+  - adding a new Illusive-Incident-Escalation playbook
```
Comment (guyfreund) [inline Packs/IllusiveNetworks/ReleaseNotes/1_0_2.md:4]:
```suggestion
  - Added a new Illusive-Incident-Escalation playbook
```
Please change all to past tense.

**Comment by adiozer** [comment]
@guyfreund This PR can be closed. won't be merged.

File: Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py [hunk 7]
```diff
@@ -363,6 +382,8 @@ def get_forensics_timeline_command(client: Client, args: dict) -> Tuple:
 
     try:
         result = client.get_forensics_timeline(incident_id, start_date, end_date)
+        for evidence in result:
+            evidence['date'] = evidence.get('details').get('date')
```
Comment (guyfreund) [inline Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py:386]:
```suggestion
            evidence['date'] = evidence.get('details', {}).get('date')
```

**Comment by guyfreund** [comment]
Closing, as @adiozer said...

**Pull request closed without merging**